### ISSUE TITLE
How to load pre-trained weights for a custom backbone?

### ISSUE THREAD

**Issue by prk-vinay** [open-mmlab/mmdetection#5215]
I want to use a custom backbone (LambdaResNet) for instance segmentation task (Cascade Mask RCNN). By reading the documentation, I was able to use the custom backbone. However, I am not able to load its pre-trained weights (I have its pretrained weights in .pth file). Is there any way to do this?

**Comment by hhaAndroid** [comment]
Hi @prk-vinay You can refer to the following template

model: 
```python
@BACKBONES.register_module()
class LambdaResNet(BaseModule):
    def __init__(...,
                         pretrained=None,
                          init_cfg=None)):
         super(LambdaResNet, self).__init__(init_cfg)
         self.init_cfg = dict(type='Pretrained', checkpoint=pretrained)
          ....
```

config:

```python
model = dict(
    type='xxx',
    pretrained='you pre-trained weights',
```
